.travis.yml

```diff
@@ -11,14 +11,11 @@ cache:
   - $GOCACHE
   - $GOPATH/pkg/mod
 
-services:
-- docker
-
 jobs:
   include:
   - name: Check generated files
-    script: make --always-make generate-in-docker && git diff --exit-code
+    script: make --always-make generate && git diff --exit-code
   - name: Run tests
-    script: make --always-make test-in-docker
```

Comment:
This needs to be updated https://github.com/coreos/kube-prometheus#contributing but I am sure there are a couple of other places.

Reply:
Good catch, I forgot to remove the occurrence of `test-in-docker` and `generate-in-docker` in the documentation

Reply:
It was actually the only place.